openssl/src/ssl/mod.rs

```diff
@@ -956,7 +956,8 @@ impl SslContextBuilder {
 
     /// Sets the leaf certificate.
     ///
-    /// Use `add_extra_chain_cert` to add the remainder of the certificate chain.
+    /// This chain should contain all certificates necessary to go from the certificate specified by
```

Comment:
This method doesn't take a chain, it takes a single certificate. What is wrong about the current documentation?

Reply:
my bad, ignore this change